File: Slice the middle of a list backwards.js
function reverseMiddle(array) {
  let even = array.slice(Math.floor(array.length/2)-1,Math.floor(array.length/2)+1);
  let odd =  array.slice(Math.floor(array.length/2)-1,Math.floor(array.length/2)+2);
  even.reverse();
  odd.reverse();
	return array.length%2===0 ?  even : odd;
}

File: Slice the middle of a list backwards/Slice the middle of a list backwards.py
def reverse_middle(lst):
    even = lst[len(lst)//2-1:len(lst)//2+1][::-1]
    odd = lst[len(lst)//2-1:len(lst)//2+2][::-1]
    return even if len(lst)%2==0 else odd
